Sources/Vapor/HTTP/Server/HTTPServerHandler.swift

```
@@ -59,6 +59,10 @@ final class HTTPServerHandler: ChannelInboundHandler, RemovableChannelHandler {
                 case .failure(let error):
                     if case .stream(let stream) = response.body.storage {
                         stream.callback(ErrorBodyStreamWriter(eventLoop: request.eventLoop, error: error))
+                    } else if case .asyncStream(let stream) = response.body.storage {
+                        Task {
+                            try await stream.callback(ErrorBodyStreamWriter(eventLoop: request.eventLoop, error: error))
```

Probably better to use `try?` here rather than let errors propagate, we're already in an error-handling path as it is. (Yes, I realize it doesn't really matter with this writer, but as a matter of good form.)